### PR TITLE
v4.2.0

### PR DESCRIPTION
**Pull request by LucianoPAlmeida**
## Checklist

- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [ ] New extensions are written in Swift 4.
- [ ] New extensions support iOS 8.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+.
- [ ] I have added tests for new extensions, and they passed.
- [ ] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [ ] All extensions are declared as **public**.
- [ ] I have added a changelog entry describing my changes.

### CODE REVIEW

**Comment by LucianoPAlmeida** [comment]
@SD10 Sorry, I notice this time that my upstream remote is still configured to my fork 😂
I think I'm going to delete it after this one hahaha

**Comment by LucianoPAlmeida** [comment]
@omaralbeik Sure, merged :)) 
About podspec you mean s.version = '4.2.0' on the file?

**Comment by omaralbeik** [comment]
Exactly @LucianoPAlmeida 👍

**Comment by LucianoPAlmeida** [comment]
@omaralbeik I remember doing that :)) But sure will check

File: SwifterSwift.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name = 'SwifterSwift'
-  s.version = '4.1.1'
+  s.version = '4.2.0'
```
Comment (LucianoPAlmeida) [inline SwifterSwift.podspec]: @omaralbeik Done 👍

**Pull request merged**